sklearn/utils/testing.py

```diff
@@ -970,3 +975,61 @@ def check_docstring_parameters(func, doc=None, ignore=None, class_name=None):
             if n1 != n2:
                 incorrect += [func_name + ' ' + n1 + ' != ' + n2]
     return incorrect
+
+
+def assert_run_python_script(source_code, timeout=60):
+    """Utility to check assertions in an independent Python subprocess.
+
+    The script provided in the source code should return 0 and not print
+    anything on stderr or stdout.
+
+    This is a port from cloudpickle https://github.com/cloudpipe/cloudpickle
+
+    Parameters
+    ----------
+    source_code : str
+        The Python source code to execute.
+    timeout : int
+        Time in seconds before timeout.
+    """
+    def _make_cwd_env():
+        """Helper to prepare environment for the child processes"""
+        sklearn_repo_folder = op.normpath(
+            op.join(op.dirname(__file__), '../..'))
```

Comment:
Argl this is the cause of the coverage issue: `sklearn_repo_folder` will be used as CWD (current working directory) for the subcommand.

The way we compute it will be dependent on `__file__`.

To make this independent of the location of the helper function, please use the parent folder of `sklearn.__file__` instead of the grand parent folder of  `__file__` which would be location dependent:

```python
sklearn_repo_folder = op.normpath(
            op.join(op.dirname(sklearn.__file__), '..'))
```

Reply:
I've changed the `cwd` and it shows the repo folder as expected but we still have the same errors :/